mock/py/mockbuild/plugins/root_cache.py

```diff
@@ -150,6 +180,16 @@ def _unpack_root_cache(self):
                 )
                 for item in self.exclude_dirs:
                     mockbuild.util.mkdirIfAbsent(self.buildroot.make_chroot_path(item))
+
+                if self.buildroot.is_bootstrap and self.buildroot.use_bootstrap_image:
+                     # The bootstrap image frequently lacks distribution-gpg-keys.
+                     # Copy the files from the host to avoid invoking package manager
+                     # or rebuilding the cached bootstrap chroot.
+                     keys_path = "/usr/share/distribution-gpg-keys"
+                     getLog().debug("Copying '%s' to the bootstrap chroot", keys_path)
+                     cmd = ["cp", "-a", keys_path, self.buildroot.make_chroot_path(keys_path)]
+                     mockbuild.util.do(cmd)
```

Comment:
This is basically https://github.com/rpm-software-management/mock/issues/308 but not bind-mount, and into bootstrap chroot only.

Just saying it would be nice if we could have this done even without `self.buildroot.is_bootstap_image` condition, but it can be done later.  Also note that bind mount would be slightly nicer I guess.